Dockerfile

```diff
@@ -5,4 +5,5 @@ ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1
 RUN echo "deb http://ppa.launchpad.net/ansible/ansible/ubuntu trusty main" >> /etc/apt/sources.list && \
 	apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 93C4A3FD7BB9C367 && \
 	apt-get update && \
-	apt-get install -y ansible=2.5\*
+	apt-get install -y --no-install-recommends ansible=2.7\* && \
```

Comment:
I was curious which packages would be left out by this:

```
  Recommends: python-jmespath
  Recommends: python-kerberos
  Recommends: python-libcloud
  Recommends: python-selinux
  Recommends: <python-winrm>
  Recommends: python-xmltodict
```

I don't see how we would need any of them, so all good!